scripts/dev_vm.sh

```diff
@@ -0,0 +1,299 @@
+#!/usr/bin/env bash
+
+##
+## Script to create a development VM on GCP
+##
+## Options:
+##  --delete             delete the VM
+##  --docker             install docker
+##  --k3s                install k3s
+##  --k9s                install k9s
+##  --kubeconfig         copy the remote kubeconfig locally
+##  --kubeconfig-path    the path to copy the kubeconfig to (defaults to ~/.kube)
+##  --kubeconfig-update  overwrite the local default kubeconfig with the remote one
+##  --kurl               install the latest kurl
+##  --name               VM name (default: $(whoami)-dev)
+##  --ssh                ssh into the VM
+##  --update             update all packages on the VM
+##  -v | --verbose  print script commands
+##
+
+set -e
+
+# GCP VM details, change to suit your needs
+VM_REGION="us-west3"
+VM_PROJECT="replicated-qa"
+VM_TYPE="n1-standard-8"
+VM_DISK_SIZE="500GB"
```

Comment:
can we make the default smaller? KOTS doesn't need that much disk space. i'd suggest `50GB`

Reply:
done, I originally chose 500GB because the gcp docs show a 10x increase in IOPs when going from 50 -> 500, but I never did any tests to confirm the performance difference